src/backends/backend/triton_model.cc

```diff
@@ -150,8 +150,25 @@ TritonModel::Create(
   RETURN_IF_ERROR(LocalizeDirectory(
       JoinPath({model_repository_path, model_name}), &localized_model_dir));
 
-  const std::string backend_libname =
-      "libtriton_" + model_config.backend() + ".so";
+  std::string backend_libname = "libtriton_" + model_config.backend() + ".so";
+  if (model_config.backend() == "tensorflow") {
```

Comment:
You'll have to adjust the backend name used for the global path below as well. So that with /opt/tritonserver/backends/tensorflow1 or /opt/tritonserver/backends/tensorflow2 is used.